app/models/reports/schema_v2.rb

```diff
@@ -226,5 +186,38 @@ def percentage(numerator, denominator)
       return 0 if numerator.nil? || denominator.nil? || denominator == 0 || numerator == 0
       ((numerator.to_f / denominator) * 100).round(PERCENTAGE_PRECISION)
     end
+
+    # Grab a particular summed field for a region. We return data in the format of:
+    #   { region_slug => { period_1 => x, period_2 => y }, ...}
+    # to maintain consistency w/ the format callers expect from the Repository.
+    #
+    # Note that we do filtering on the result set to limit the returned amount of data to the data that callers are
+    # requesting via the `periods` argument the Repository was created with.
+    def sum(region, field)
+      summed_field = "sum_#{field}"
+      facility_state_data(region)
+        .reject { |facility_state| Period.month(facility_state.month_date) < earliest_patient_recorded_at_period[region.slug] }
+        .select { |facility_state| facility_state.period_month_date.in?(periods) }
+        .to_h { |facility_state| [Period.month(facility_state.month_date), facility_state.public_send(summed_field)] }
+        .tap { |hsh| hsh.default = 0 }
+    end
+
+    delegate :sql, to: Arel
```

Comment:
Do we need this since we're doing `Arel.sql` and it's not too verbose?

Reply:
true, this was leftover - removed